internal/go/skycfg/proto_api.go

```diff
@@ -236,6 +238,30 @@ func fnProtoToJson(t *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple
 	return starlark.String(jsonData), nil
 }
 
+// Implementation of the `proto.to_any()` built-in function. Returns a
+// skyProtoMessage with an `Any` proto.Message in it.
+func fnProtoToAny(t *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var msg *skyProtoMessage
+	err := wantSingleProtoMessage(fn.Name(), args, kwargs, &msg)
+	if err != nil {
+		return nil, err
+	}
+
+	// Disambiguate between golang encoded proto messages.
```

Comment:
I'm not sure why this branch is needed. Why not `any, err := ptypes.MarshalAny(msg.msg)`? Was there a case you encountered where a protobuf message has no type name?

Reply:
I think this is a holdover/cargo-cult from the first iteration of this change. I'll remove it.